Scripts/ImportVirtualMachines/ImportVirtualMachines.ps1

```diff
@@ -164,7 +164,7 @@ try {
     else {
 
         # We need to copy all the VMs in the lab
-        [array] $sourceResourceIds = (Get-AzureRmResource -ResourceType "Microsoft.DevTestLab/labs/virtualmachines" -ResourceGroupName $sourceLab.ResourceGroupName).ResourceId
+        [array] $sourceResourceIds = (Get-AzureRmResource -ResourceType "Microsoft.DevTestLab/labs/virtualmachines" -ResourceGroupName $sourceLab.ResourceGroupName | Where-Object {$_.ResourceName -like "$SourceDevTestLabName/*" }).ResourceId
```

Comment:
This is a fix recommended from a customer & Leo V., we were accidently picking up the case where we have multiple DevTest Labs in the same resource group.